### PR TITLE
Revise api doc

### PR DESCRIPTION
**Pull request by mattopolitan**
Added "extra_data" field submission in api createTransaction

### CODE REVIEW

File: docs/api/http.md [hunk 1]
```diff
@@ -79,7 +79,8 @@ curl -X POST http://127.0.0.1:8090/wallet/createtransaction  -d
     "owner_address_": "TRGhNNfnmgLegT4zHNjEqDSADjgmnHvubJ",
     "to_address_": "TJCnKsPa7y5okkXvQAidZBzqx3QyQ6sxMW",
     "amount": 1000000,
-    "visible": true
+    "visible": true,
+    "extra_data": "You may write your remarks here."
```
Comment (andelf) [inline docs/api/http.md:83]: DO TO mention the `base64` encoding.

**Comment by andelf** [comment]
DO NOT add extra spaces before line break.
Review your diffs.

File: docs/api/http.md [hunk 2]
```diff
@@ -1278,6 +1280,11 @@ $ curl -X POST  http://127.0.0.1:8090/wallet/getblockbylimitnext -d
     "endNum": 2
 }'
 ```
+Parameter startNum: The start block height, itself included  
+Parameter endNum: The end block height, itself not included
```
Comment (andelf) [inline docs/api/http.md:1284]: line breaks. Be consistent with above APIs.

File: docs/api/http.md [hunk 3]
```diff
@@ -1278,6 +1280,11 @@ $ curl -X POST  http://127.0.0.1:8090/wallet/getblockbylimitnext -d
     "endNum": 2
 }'
 ```
+Parameter startNum: The start block height, itself included  
+Parameter endNum: The end block height, itself not included  
+Return: The list of the blocks  
+
+- wallet/getblockbylatestnum
```
Comment (andelf) [inline docs/api/http.md:1287]: DO TO review your diffs before commit.
The above lines might come from an error some merge.

**Comment by mattopolitan** [comment]
Sorry that I've done something wrong for my git merge with the upstream. I'll do another pull request instead.

**Pull request closed without merging**